forge/test/model_demos/high_prio/cnn/pytorch/test_detr.py

```diff
@@ -0,0 +1,69 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+
+# SPDX-License-Identifier: Apache-2.0
+# Detr model having both object detection and segmentation model
+# https://huggingface.co/docs/transformers/en/model_doc/detr
+
+import pytest
+import forge
+import requests
+from torchvision import transforms
+from transformers import (
+    DetrForObjectDetection,
+    DetrImageProcessor,
+    DetrConfig,
+    DetrFeatureExtractor,
+    DetrForSegmentation,
+)
+from PIL import Image
+
+detection_variants = ["facebook/detr-resnet-50"]
+segmentation_variants = ["facebook/detr-resnet-50-panoptic"]
+
+# Load the image
+url = "http://images.cocodataset.org/val2017/000000397133.jpg"
+image = Image.open(requests.get(url, stream=True).raw).convert("RGB")
+
+
+@pytest.mark.parametrize("variant", detection_variants)
+def test_detr_detection(variant):
+
+    # Load the model
+    model = DetrForObjectDetection.from_pretrained(variant)
+    processor = DetrImageProcessor.from_pretrained(variant)
+    model_config = DetrConfig.from_pretrained(variant)
```

Comment:
model should be loaded in one of the following ways:
- with `.from_pretrained` as you did
- with `config` for example: `model = DetrForObjectDetection(model_config)`

As you choose the first one, there is probably no need for `model_config`
Also if that is the case, remove the `DetrConfig` from import as well :) 

One more thing, everything we are discussing here can be found in the official documentation on [huggingface](https://huggingface.co/docs/transformers/en/model_doc/detr) :)